braid/braid.h

```diff
@@ -57,15 +57,15 @@ extern "C" {
  */
 
 /** Define Fortran name-mangling schema, there are four supported options, see braid_F90_iface.c */
-#define braid_FMANGLE 1
+#define braid_FMANGLE 0
 /** Turn on the optional user-defined spatial coarsening and refinement functions */ 
 #define braid_Fortran_SpatialCoarsen 0
 /** Turn on the optional user-defined residual function */ 
-#define braid_Fortran_Residual 1
```

Comment:
I had to set these all to zero in order to build libbraid.so. I don't really understand why, but the F90 interface seems to interfere with building the shared library. I would like to fix this!

Reply:
I don't understand this either...I'm not sure if we want to turn off the FORTRAN interface, or leave instructions for julia users to do this

Reply:
Ok, I think I have this figured out. I guess clang doesn't automatically do a dynamic lookup for undefined symbols, so on my machine I need to add the flag -undefined dynamic_lookup. This fixes the error, so thankfully it seems it was just a 'me' problem.

Reply:
This is now fixed!